kubeflow/weaveflux/prototypes/weaveflux.jsonnet

```diff
@@ -1,10 +1,10 @@
-// @apiVersion 0.1
+// @apiVersion 0.1.1
 // @name io.ksonnet.pkg.weaveflux
 // @description Weaveworks Flux integration with Kubeflow
 // @shortDescription A Flux meets Kubeflow
 // @param name string Name to give to each of the components
 // @optionalParam giturl string null Your default git URL.
-// @optionalParam serviceType string ClusterIP The service type for WeaveFlux.
+// @optionalParam serviceType string LoadBalancer The service type for WeaveFlux.
```

Comment:
Please don't do this. Creating a LoadBalancer by  default is opening up an external port. We do not want our default configs to be opening up external access by default. 
Our default operation should always be secure; even if that means telling people to do inconvenient things like use kubectl port-forward.

On GCP we should really try to make this secure using IAP.

Reply:
How would you feel about putting it behind ambassador or envoy or something similar? I feel like that's a more automated yet secure method. making a user port-forward isn't the end of the world but I would like to automate it and make it secure.

Reply:
Went ahead and did the NodePort. I have a long-term plan of using Ambassador and will include it in the v0.2 of WeaveFlux with the CLI tool.